File: Go/fizz_buzz.go
package main

import "fmt"

func check(num1 int, num2 int) bool {
	if num1 % num2 == 0 {
		return true
	}
	return false
}

func main() {
	start := 1;
	max := 1000;

	for start <= max {
		s,n := check(start,3),check(start,5)
		if s == true && n == true {
			fmt.Println("FizzBuzz")
		} else if s == true && n == false {
			fmt.Println("Fizz")
		} else if s == false && n == true {
			fmt.Println("Buzz")
		} else {
			fmt.Println(start)
		}

		start ++
	}
}

File: Dart/fizzbuzz.dart
import 'dart:io';

void main() {
  var number = 100;

  for (var i = 0; i < number; i++){

    if ( i % 3 == 0 && i % 5 == 0){
      print('fizz buzz');
    }
    else if (i % 3 == 0){
      print('fizz');
    }
    else if (i % 5 == 0 ){
      print('buzz');
    }
    else{
      print(i);
    }
  }

} 